lib/rootApi.js

```diff
@@ -94,31 +95,20 @@ api.add = callbackify(async (actor, options) => {
     }
   }
 
-  // FIXME: generalize configuration validation and call generalized method
-  if(!(ledgerConfiguration && jsonld.hasValue(
-    ledgerConfiguration, 'type', 'WebLedgerConfiguration'))) {
-    throw new BedrockError(
-      'The provided configuration event is malformed. ' +
-      'An object with `type` set to `WebLedgerConfiguration` does not exist.',
-      'DataError',
-      {httpStatusCode: 400, public: true, ledgerConfiguration});
-  }
+  // perform basic schema validation
+  validator.validateGenesisLedgerConfiguration({ledgerConfiguration});
```

Comment:
Does this need `await`?

Reply:
It looks like it currently doesn't, but I recommend we make it so.

Reply:
done.